Add websocket tutorial (#1101)
timescale/docs: getting-started/next-steps.md

Look at the [how-to guide on Migrating Data][migrate-data] for more help and
suggestions of where to start.

## Ingest real-time data
Working with sample data can teach you a lot about TimescaleDB, but you might

> be interested to ingest market data real-time. Check out our

Reviewer: ```suggestion
like to try ingesting market data in real time. Check out our
```

> complementing tutorial

Reviewer: ```suggestion
related tutorial
```
